docs/requirements.txt

```diff
@@ -1,2 +1,4 @@
 sphinx>=1.4, !=1.5.4
 sphinx_copybutton
+alabaster
```

Comment:
Alabaster is unneeded as a dependency as both sphinx and alabaster_jupyterhub would install. @choldgraf @betatim